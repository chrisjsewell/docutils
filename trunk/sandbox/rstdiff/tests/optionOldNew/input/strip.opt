--strip-elements-with-class=both --old --strip-comments --strip-elements-with-class=old --new --strip-elements-with-class=new --both --strip-elements-with-class=all
