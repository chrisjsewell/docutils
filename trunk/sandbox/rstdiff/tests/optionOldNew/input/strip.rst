.. class:: new

   not in new

.. class:: old

   NOT IN OLD

.. class:: both

   Never

.. class:: all

   Nowhere

.. Some comment
