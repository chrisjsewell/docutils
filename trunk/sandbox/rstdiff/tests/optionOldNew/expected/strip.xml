<?xml version="1.0" encoding="utf-8"?>
<!-- Generated by Docutils 0.8 -->
<document source="input/strip.rst =&gt; input/strip.rst">
    <paragraph classes="new change-replaced">
        not in new
    </paragraph>
    <paragraph classes="old change-replacement">
        NOT IN OLD
    </paragraph>
    <comment classes="change-added" xml:space="preserve">
        Some comment
    </comment>
</document>
